google/gcloud/config.go

```diff
@@ -1,55 +1,37 @@
 package gcloud
 
+import (
+	"github.com/pkg/errors"
+)
+
 type Config struct {
-	ConfigDir    string        `json:"configDir" yaml:"configDir"`
-	Environments []Environment `json:"environments" yaml:"environments"`
+	Login           bool          `json:"login" yaml:"login"`
```

Comment:
This is a little unclear, why have the login bool flag? It's not going to log in just offers more commands.

Reply:
Perhaps UseDefaultAccount or something?

Reply:
completely different now ... we define `Clusters`